discojs/discojs-core/src/client/base.ts

```diff
@@ -52,6 +52,7 @@ export abstract class Base {
   abstract onRoundEndCommunication (
     updatedWeights: WeightsContainer,
     staleWeights: WeightsContainer,
+    updatedMomentum: WeightsContainer,
```

Comment:
not sure if we need to send around global momentum. better to only use local momentum to determine the clipping centers

Reply:
this is basically the `m_i` in the formula (i.e.: the momentum computed by the client `i`) which is then sent to the server. It's not the global momentum, but refers to a specific local client.

Reply:
so i think this one doesn't need to be sent. it's computed locally and remains on the client, no?

Reply:
The server uses all the agents' momentum vectors to compute the global shared momentum. Thus, it needs to know all agents' values, right? At least in my implementation, I use these values on the server to first computer the centered part and then the clipping factor while computing the updated global shared momentum. Depending on whether the model use or not the Byzantine-Robust Aggregator, each client sends the updatedWeights (potentially with the privacy-preserving computations) or updatedMomentum.